.buildkite/pipeline.package.mbp.yml

```diff
@@ -61,6 +107,22 @@ steps:
       - step: "package-arm-snapshot"
         allow_failure: false
 
+  - label: "DRA FIPS snapshot"
```

Comment:
I believe this duplicates what has already been uploaded by the normal `DRA snapshot ` since all the artifacts get uploaded to a GS bucket before being downloaded by the `DRA*` steps and uploaded using release-manager.

A better pattern would be to make `DRA snapshot` wait before fips and non-fips packaging steps have completed and then download all artifacts at once before invoking release-manager.

The same thing applies to `DRA staging` flow